packages/@aws-cdk/cloud-assembly-schema/lib/cloud-assembly/schema.ts

```diff
@@ -60,6 +60,14 @@ export interface MissingContext {
    * A set of provider-specific options.
    */
   readonly props: ContextQueryProperties;
+
+  /**
+   * The ARN of the role that should be used to look up
+   * the missing values in this context.
+   *
+   * @default - None
+   */
+  readonly lookupRoleArn?: string;
```

Comment:
I wonder if this makes more sense inside of `props`?

Reply:
Yes, good point.